apps/server/src/socket.ts

```diff
@@ -42,14 +44,20 @@ const onOfferRide =
     redis.publish('driverLocationUpdate', '');
   };
 
-const onFindRide = (socket: Socket) => async (rideId: string) => {
+const onFindRide = (socket: Socket) => async (rideId: string, preferredVehicle: VehicleType[], passengers: number) => {
```

Comment:
couldn't we get `preferredVehicle` and `passengers` from `ride` [below](https://github.com/georgevmlambton/location-destination/pull/21/files#diff-16468b5468b9d9131700781f197fe79eb8de457ac4c00be6b106cea75e315710L53)?
```ts
  const ride = await Ride.findById(rideId);
```

We won't need to pass extra args from the frontend then. We just pass `rideId`

Reply:
Fixed and commited.